tests/e2e/cucumber/environment/index.ts

```diff
@@ -68,11 +68,10 @@ Before(async function (this: World, { pickle }: ITestCaseHookParameter) {
     }
   })
   if (!config.basicAuth) {
```

Comment:
do we still use this ?

Reply:
Maybe. We still have code flow for basic auth
https://github.com/owncloud/web/blob/89c7d5ba3a544ff7471113a0e81b2a5046ea3faa/tests/e2e/support/api/http.ts#L9-L17

Reply:
In the next PR please remove **basicAuth** related code if we are not using it. If it's a dead/orphan code then it is better to remove it completely if it remains as orphan code then in future we may face flakiness or hard-to-decode test code flow 
```
   const authHeader = { 
     Authorization: 'Basic ' + Buffer.from(user.id + ':' + user.password).toString('base64') 
   } 
```

Reply:
Sure thanks. I will try to look into the code and if it's not used anymore, it could be thrown out

Reply:
@ScharfViktor do we want to remove basicAuth setup for e2e test?

Reply:
@S-Panta I don't think we will remove support for basicAuth so please don't bother removing it

Reply:
> @ScharfViktor do we want to remove basicAuth setup for e2e test?

I'd leave it as an option